operator/check_operator.go

```diff
@@ -91,6 +72,35 @@ func (c operatorCheck) Run(ctx context.Context) (certification.Results, error) {
 	return eng.Results(ctx), nil
 }
 
+// List the available operator checks.
+func (c operatorCheck) List(ctx context.Context) (policy.Policy, []check.Check, error) {
+	switch {
+	case c.image == "":
+		return policy.PolicyNone, []check.Check{}, preflighterr.ErrImageEmpty
```

Comment:
See comment about type alias. Return "" for policy on error. It shouldn't _look_ like a valid policy. And if err != nil, none of the other values returned should be expected to be valid.

Reply:
fixed